src/main/java/frc/robot/RobotContainer.java

```diff
@@ -6,11 +6,8 @@
 
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
-import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
```

Comment:
Go ahead and leave existing controllers in there, just don't use them if not needed.